website/docs/reference/global-configs.md

```diff
@@ -247,21 +263,34 @@ config:
 
 </File>
 
-## Experimental parser
-With the `USE_EXPERIMENTAL_PARSER` config, you can opt into the latest and greatest experimental version of the static parser, which is still being sampled for 100% correctness. See [the docs on parsing](parsing#experimental-parser) for more details.
+### Strict
+
+As of v1.0, the `-S` or `--strict` flag has been deprecated.
+
+<VersionBlock firstVersion="1.1">
+### Suppress non-error logs in output
+
+By default, dbt shows all logs in standard out (stdout). You can use the `QUIET` config to show only error logs in stdout. For example, you might narrow down your logs to error logs only because it might help you more easily find and debug a jinja error.
```

Comment:
The reason someone would use `QUIET` is to:
- suppress all logs except errors
- still include the output of anything passed to the `print()` macro (#1320)

Reply:
A community contributor also just added a `NO_PRINT` config / `--no-print` flag: https://github.com/dbt-labs/dbt-core/pull/4854

It would make sense to document these two alongside each other, or link between them.

Reply:
Addressed these in https://github.com/dbt-labs/docs.getdbt.com/pull/1319/commits/af4adbe7e555faf316fb6b9fa174a8419d18bc5e